MonoGame.Framework/Platform/Media/Song.Android.cs

```diff
@@ -183,11 +178,6 @@ private bool PlatformIsRated()
             return false;
         }
 
-        private string PlatformGetName()
```

Comment:
Removed these as well.

For all platforms it defaults to the file name unless the platform code changes it during Song initialization.